topology/probes/k8s/ingress.go

```diff
@@ -89,12 +148,41 @@ func (p *ingressProbe) OnDelete(obj interface{}) {
 	}
 }
 
+func (p *ingressProbe) onNodeUpdated(srvNode *graph.Node) {
```

Comment:
Shall we not need to remove the link to service on OnNodeDeleted() event ?

Reply:
delNode func within `topology/graph/graph.go` already automatically deletes all edges